.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
+    "env",
+    "react"
+  ]
+}
```

Comment:
Missing newline at end of file

Reply:
```

```

Reply:
Thanks for noticing the missing newline!